.github/workflows/playwright_pre_daily.yml

```diff
@@ -1,4 +1,4 @@
-name: Penpot Regression Tests on PRE env
+name: Daily Penpot Regression Tests on PRE
```

Comment:
Why you deleted "Daily" in the name? We change this, adding "Daily", because is more descriptive in the reports.

Reply:
It should be with the word Daily and I see it like that locally. I'll ensure to push it with the right name.

Reply:
The file is right, read it again. I'm ADDING Daily to help identifying the workflow, not the opposit. It was bad.